src/SprykerEco/Zed/Heidelpay/Persistence/Propel/Schema/spy_heidelpay.schema.xml

```diff
@@ -11,20 +11,20 @@
         <column name="id_payment_heidelpay" required="true" type="INTEGER" autoIncrement="true" primaryKey="true"/>
         <column name="fk_sales_order" required="true" type="INTEGER"/>
 
-        <column name="id_payment_reference" required="false" type="VARCHAR"/>
-        <column name="id_payment_registration" required="false" type="VARCHAR"/>
-        <column name="id_basket" required="false" type="VARCHAR"/>
+        <column name="id_payment_reference" required="false" type="VARCHAR" size="255"/>
+        <column name="id_payment_registration" required="false" type="VARCHAR" size="255"/>
+        <column name="id_basket" required="false" type="VARCHAR" size="255"/>
 
-        <column name="payment_method" required="true" type="VARCHAR"/>
+        <column name="payment_method" required="true" type="VARCHAR" size="255"/>
 
-        <column name="easy_credit_fee" required="false" type="INTEGER" />
+        <column name="easy_credit_fee" required="false" type="INTEGER"/>
 
         <column name="date_of_birth" required="false" type="DATE"/>
-        <column name="salutation" required="false" type="VARCHAR"/>
+        <column name="salutation" required="false" type="VARCHAR" size="255"/>
 
         <column name="connector_invoice_account_info" required="false" type="LONGVARCHAR" />
 
-        <foreign-key name="spy_payment_payone-fk_sales_order" foreignTable="spy_sales_order" phpName="SpySalesOrder">
+        <foreign-key name="spy_payment_heidelpay-fk_sales_order" foreignTable="spy_sales_order" phpName="SpySalesOrder">
```

Comment:
It was decided to release it as a minor, because a foreign-key constraint is a database object as well, and thus its name needs to be unique. In the current situation, we have an intersection with the Payone payment provider.